docker/importer/importer.py

```diff
@@ -754,14 +791,13 @@ def _process_updates_rest(self, source_repo: osv.SourceRepository):
       except Exception as e:
         logging.exception('Failed to parse %s: error type: %s, details: %s',
                           vuln.id, e.__class__.__name__, e)
-        import_failure_logs.append('Failed to parse vulnerability "' + vuln.id +
-                                   '"')
+        import_failure_logs.append(f'Failed to parse vulnerability "{vuln.id}"')
         continue
 
     replace_importer_log(storage.Client(), source_repo.name,
                          self._public_log_bucket, import_failure_logs)
 
-    source_repo.last_update_date = import_time_now
+    source_repo.last_update_date = request_last_modified or vulns_last_modified
```

Comment:
Maybe use `max` here?

Reply:
You can't do a `max` comparison with a `None` and a `datetime` if `request_last_modified` isn't set.

But also, I want it to be consistent with which time it uses, preferring the server's `Last-Modifed` time since it's check first for early exit.